utils/solutions_from_games.txt

```diff
@@ -0,0 +1,249 @@
+# TMOU 24 2023
+POLEDNE KILAK
+OHNIŠTĚ ŠEDESÁT METRÚ JJZ OD VYHLÍDKY KAMENNÁ KOLONIE
+ZIVLY
+PAINTBALL PARK
+STUDNA ROLNICKA
+VYSILAC J
+HRISTE TICHA
+HRBITOV J ROH EL SLOUP
+KOTA DVA KRAT I
+SMRK MODRA
+Vrchol JV od Peregr kaple
+BUS STOP U MLYNA SZ
+ZVON
+KRIZ U STAJE
+OBECNI MUZEUM
+BEZTE DO SOKOLA HESLO GRAM
+OLYMP
+# Palapeli 2024
+CDXV
+KONSEL
+BEZPREKVAPENINAROZHLEDNU
```

Comment:
Proč jsou v některých řešeních mezery mezi slovy a v některých ne?

Reply:
Protože to někteří orgová uvádějí tak a někteří jinak a mně časem klesala chuť to ještě editovat.

Reply:
Pak mi dává smysl to zmínit někde v tom souboru na začátku